ts/packages/ts/src/stores/Configu.ts

```diff
@@ -9,8 +9,22 @@ export type ConfiguConfigStoreConfiguration = {
   source?: string;
 };
 
+const ConfiguConfigStoreApprovalQueueErrorName = 'ConfiguConfigStoreApprovalQueueError';
```

Comment:
@RichardAkman sorry but I still cant accept this change, its too wired, your only option is to use instanceOf if your going with the Custom Error class.